File: app/src/main/java/com/happs/ximand/ringcontrol/model/object/exception/BluetoothIsDisabledException.java
package com.happs.ximand.ringcontrol.model.object.exception;

import com.happs.ximand.ringcontrol.R;

@Deprecated
public class BluetoothIsDisabledException extends BluetoothExceptionDep {

    @Override
    public int getMessageResId() {
        return R.string.bluetooth_is_disabled;
    }
}

File: app/src/main/java/com/happs/ximand/ringcontrol/model/bl/exception/BluetoothIsDisabledException.kt
package com.happs.ximand.ringcontrol.model.bl.exception

import com.happs.ximand.ringcontrol.R
import com.happs.ximand.ringcontrol.model.`object`.exception.BaseException

class BluetoothIsDisabledException : BaseException(CODE) {

    companion object {
        const val CODE = 0x20
    }

    override val descriptionResId: Int = R.string.bluetooth_is_disabled

}